go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/GoogleContainerTools/kpt v1.0.0-beta.29.0.20230327202912-01513604feaa
 	github.com/GoogleContainerTools/kpt-functions-sdk/go/fn v0.0.0-20230302070146-e8e9cb3c3ae2
 	github.com/google/go-cmp v0.5.9
+	github.com/nephio-project/nephio-controller-poc v0.0.2
```

Comment:
If we are grabbing a type from here, we should move it to nephio-project/api instead.